src/gui/Src/Gui/HandlesView.cpp

```diff
@@ -317,11 +321,26 @@ void HandlesView::disableWindowSlot()
     enumWindows();
 }
 
+
 void HandlesView::followInDisasmSlot()
 {
     DbgCmdExec(QString("disasm %1").arg(mWindowsTable->mCurList->getCellContent(mWindowsTable->mCurList->getInitialSelection(), 0)));
 }
 
+void HandlesView::followInThreads()
+{
+    QString threadID = mWindowsTable->mCurList->getCellContent(mWindowsTable->mCurList->getInitialSelection(), 4);

```

Comment:
Here you can use `getCellUserdata` to store a `duint` (thread id) for a column. Just have to populate it when the handles are being refreshed.

Reply:
Where can I find the `getCellUserData`?

Maybe I could do the same approach as CallStackView: {Thread Name} - {Thread Id}

Reply:
Didn't forget about this, locally I did the plumbing to add support for `getCellUserdata` to the `SearchListView` but something isn't working right and I need some time to debug it...

Reply:
Ok, in this case the `getCellUserdata` works, but the command `showthreadid` works for thread numbers in DEC (as displayed on menu title), not for HEX (as the code is putting in that way).

I will add support for HEX numbers for `showthreadid` command in another MR. 

Is somewhere else the function failing?

Reply:
Hm probably the command should use valfromstring to parse the arguments to numbers right?

Reply:
Theres a plain strtoll 😄 😄 😄 , I do not remember why I did not use it, probably I thought that it would be an overtake (we would need only numbers after all). I remember I also had troubles with HEX vs DEC. 

I could replace it with `valfromstring `.

Reply:
Yeah please do, the default radix is hex that’s why there is this ToHexPtr function to format the numbers for commands

Reply:
Done